src/Cassandra/Data/Linq/CqlExpressionVisitor.cs

```diff
@@ -72,15 +86,19 @@ internal class CqlExpressionVisitor : ExpressionVisitor
         private int _limit;
         private readonly List<Tuple<PocoColumn, object, ExpressionType>> _projections = new List<Tuple<PocoColumn, object, ExpressionType>>();
         private readonly List<Tuple<string, bool>> _orderBy = new List<Tuple<string, bool>>();
-        private readonly List<string> _selectFields = new List<string>();
+        private readonly List<string> _selectFields = new List<string>(DefaultClauseParameterCapacity);
```

Comment:
Nice to set an initial value, this will prevent the internal array of the list to be resized. Would make sense to use the number of columns of PocoData received at constructor of CqlExpressionVisitor instead of fixed value?